source/vibe/http/common.d

```diff
@@ -601,9 +601,15 @@ struct CookieValueMap {
 
 	inout(string)* opBinaryRight(string op)(string name) inout if(op == "in")
 	{
-		foreach(c; m_entries)
-			if( c.name == name )
-				return &c.value;
+		static if (__VERSION__ < 2066) {
+			foreach(c; m_entries)
+				if( c.name == name )
+					return &c.value;
+		} else {
+			foreach(ref c; m_entries)
+				if( c.name == name )
+					return &c.value;
+		}
```

Comment:
Hum.. ok. But then let's at least change it to this, so that the potential memory corruption bug is gone:
```
		foreach (ref c; m_entries)
			if (c.name == name) {
				static if (__VERSION__ < 2066)
					return cast(inout(string)*)&c.value;
				else
					return &c.value;
			}
```

I can also do that after the merge, though!